modules/test/playwright/tests/account-admin-web/accounts.spec.ts

```diff
@@ -137,3 +137,44 @@ test('LPD-18483 Add account email address', async ({
 		await apiHelpers.headlessAdminUser.deleteAccount(account.id);
 	}
 });
+
+test('LPD-18484 Add account website', async ({
+	accountsPage,
+	apiHelpers,
+	editAccountContactInformationPage,
+	editAccountContactPage,
+	editAccountPage,
+	editAccountWebsitePage,
+	page,
+}) => {
+	await apiHelpers.featureFlag.updateFeatureFlag('LPD-10855', true);
```

Comment:
if needed, instead of activate the FF at the begin and then deactive it at the end you can use the FF fixture. Have a look to test inside the [commerceMinicart file](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/commerce/commerceMiniCart.spec.ts)